pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -129,10 +119,11 @@ stdenv.mkDerivation rec {
     "swresample"
     "swscale"
     "jpeg"
-    "png"
+    # "png"
```

Comment:
Whats up with this?

Reply:
libpng.so has been removed from `opt/kingsoft/wps-office/office6`

Reply:
Then we can delete that line

Reply:
Should I  delete all `unvendoredLibraries` ？